src/components/TimeRemaining.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react'
+import styled from 'styled-components'
+
+export default function TimeRemaining({ activities }) {
+  return remainingHours(activities) !== 0 ? (
+    <PStyled>
+      You still have{' '}
+      {remainingHours(activities) % 1 !== 0
+        ? remainingHours(activities).toFixed(1)
+        : remainingHours(activities)}{' '}
+      {remainingHours(activities) === 1 ? 'hour' : 'hours'} to add to your day,
+      use {remainingHours(activities) === 1 ? 'it' : 'them'} wisely!
+    </PStyled>
+  ) : (
+    <PStyled>You don't have any hours remaining</PStyled>
```

Comment:
Beim User Test ist mir aufgefallen, dass wenn man ins Minus geht, werden Minus Stunden angezeigt und "You dont have any hours remaining" greift nicht. Vllt kann man eine logik hinzufügen, die sagt, es gibt nur 24h max und man kann nicht mehr hinzufügen.

<img width="347" alt="Bildschirmfoto 2020-03-08 um 10 36 32" src="https://user-images.githubusercontent.com/59569402/76160263-e4986480-6128-11ea-8f8e-1e6814f0706d.png">